x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/pipeline_select_option.tsx

```diff
@@ -52,7 +52,9 @@ export const PipelineSelectOption: React.FC<PipelineSelectOptionProps> = ({ pipe
       <EuiFlexItem>
         <EuiFlexGroup gutterSize="s" justifyContent="flexStart" alignItems="center">
           <EuiFlexItem grow={pipeline.modelType.length === 0}>
-            <EuiText size="s">{modelIdDisplay}</EuiText>
+            <EuiText size="s" color="subdued">
```

Comment:
Unrelated change: make model ID text color match with the same in other components (pipeline card / model selection list).